python/dgl/graphbolt/graph_storage/sampled_sub_graph.py

```diff
@@ -0,0 +1,156 @@
+"""Heterogeneous sampled subgraph."""
+# pylint: disable= invalid-name
+from collections.abc import Mapping
+
+
+class SampledSubGraph:
```

Comment:
Instead of making it a class, let's make it a API first, and then implementation.

Reply:
And move it to graphbolt/ top level folder.

Reply:
I'm not quite understand how to make it as an API, because this struct is needed as the result of sampling.

Reply:
See this https://github.com/dmlc/dgl/blob/master/python/dgl/graphbolt/dataset.py as an example.

Reply:
Changed to API style.